assets/app/models/Site.js

```diff
@@ -0,0 +1,23 @@
+var Backbone = require('backbone');
```

Comment:
No, I'm pretty sure it has a built in version of jQuery. I generally use this particular style ^^ when I want to set a particular version of jQuery to be used by Backbone or I already have it on the page.